decidim-proposals/db/migrate/20200120215928_move_proposal_endorsements_to_core_endorsements.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# This migration must be executed after CreateDecidimEndorsements migration in decidim-core.
+class MoveProposalEndorsementsToCoreEndorsements < ActiveRecord::Migration[5.2]
+  class ProposalEndorsement < ApplicationRecord
+    self.table_name = :decidim_proposals_proposal_endorsements
+  end
+
```

Comment:
If you have the migration you should have the class... but well I'll add it just in case someone goes back to version 0.20 and want to rollback the migration.